connectors/dune/README.md

```diff
@@ -0,0 +1,182 @@
+# Dune API Connector Example
+
+## Connector overview
+
+This connector retrieves query results from the [Dune API](https://dune.com/docs/api/) and syncs them into your Fivetran destination.  
+It allows you to execute one or multiple Dune queries, monitor their execution status, and fetch the final results into a unified destination table.
+
+The connector supports:
+- Running multiple queries sequentially.
+- Tracking execution status (`RUNNING`, `COMPLETED`, `FAILED`).
+- Rate limit handling (HTTP 429).
+- Incremental sync management using checkpointing.
+
+---
+
+## Requirements
+
+- [Supported Python versions](https://github.com/fivetran/fivetran_connector_sdk/blob/main/README.md#requirements)
+- Operating system:
+    - Windows: 10 or later (64-bit only)
+    - macOS: 13 (Ventura) or later (Apple Silicon [arm64] or Intel [x86_64])
+    - Linux: Ubuntu 20.04+, Debian 10+, or Amazon Linux 2+ (arm64 or x86_64)
+- A valid **Dune API Key**.
```

Comment:
here and across the README file- no bolding should be used except for UI elements like commands, menus, etc.